TwigExtension.php

```diff
@@ -8,9 +8,12 @@ class TwigExtension extends \Twig_Extension
 {
     private $router;
 
-    public function __construct(UrlGeneratorInterface $router)
+    private $stackTracePath;
+
+    public function __construct(UrlGeneratorInterface $router, $stackTracePath)
```

Comment:
I guess you must add the new param also in the service definition. => services.xml